src/sstruct_ls/protos.h

```diff
@@ -5,398 +5,10 @@
-HYPRE_Int HYPRE_SStructSSAMGSetPrintLevel ( HYPRE_SStructSolver solver, HYPRE_Int print_level );
-HYPRE_Int HYPRE_SStructSSAMGSetPrintFreq ( HYPRE_SStructSolver solver, HYPRE_Int print_freq );
-HYPRE_Int HYPRE_SStructSSAMGGetNumIterations ( HYPRE_SStructSolver solver,
-                                               HYPRE_Int *num_iterations );
-HYPRE_Int HYPRE_SStructSSAMGGetFinalRelativeResidualNorm ( HYPRE_SStructSolver solver,
-                                                           HYPRE_Real *norm );
-
-/* HYPRE_sstruct_sys_pfmg.c */
-HYPRE_Int HYPRE_SStructSysPFMGCreate ( MPI_Comm comm, HYPRE_SStructSolver *solver );
-HYPRE_Int HYPRE_SStructSysPFMGDestroy ( HYPRE_SStructSolver solver );
-HYPRE_Int HYPRE_SStructSysPFMGSetup ( HYPRE_SStructSolver solver, HYPRE_SStructMatrix A,
-                                      HYPRE_SStructVector b, HYPRE_SStructVector x );
-HYPRE_Int HYPRE_SStructSysPFMGSolve ( HYPRE_SStructSolver solver, HYPRE_SStructMatrix A,
-                                      HYPRE_SStructVector b, HYPRE_SStructVector x );
-HYPRE_Int HYPRE_SStructSysPFMGSetTol ( HYPRE_SStructSolver solver, HYPRE_Real tol );
-HYPRE_Int HYPRE_SStructSysPFMGSetMaxIter ( HYPRE_SStructSolver solver, HYPRE_Int max_iter );
-HYPRE_Int HYPRE_SStructSysPFMGSetRelChange ( HYPRE_SStructSolver solver, HYPRE_Int rel_change );
-HYPRE_Int HYPRE_SStructSysPFMGSetZeroGuess ( HYPRE_SStructSolver solver );
-HYPRE_Int HYPRE_SStructSysPFMGSetNonZeroGuess ( HYPRE_SStructSolver solver );
-HYPRE_Int HYPRE_SStructSysPFMGSetRelaxType ( HYPRE_SStructSolver solver, HYPRE_Int relax_type );
-HYPRE_Int HYPRE_SStructSysPFMGSetJacobiWeight ( HYPRE_SStructSolver solver, HYPRE_Real weight );
-HYPRE_Int HYPRE_SStructSysPFMGSetNumPreRelax ( HYPRE_SStructSolver solver,
-                                               HYPRE_Int num_pre_relax );
-HYPRE_Int HYPRE_SStructSysPFMGSetNumPostRelax ( HYPRE_SStructSolver solver,
-                                                HYPRE_Int num_post_relax );
-HYPRE_Int HYPRE_SStructSysPFMGSetSkipRelax ( HYPRE_SStructSolver solver, HYPRE_Int skip_relax );
-HYPRE_Int HYPRE_SStructSysPFMGSetDxyz ( HYPRE_SStructSolver solver, HYPRE_Real *dxyz );
-HYPRE_Int HYPRE_SStructSysPFMGSetLogging ( HYPRE_SStructSolver solver, HYPRE_Int logging );
-HYPRE_Int HYPRE_SStructSysPFMGSetPrintLevel ( HYPRE_SStructSolver solver, HYPRE_Int print_level );
-HYPRE_Int HYPRE_SStructSysPFMGGetNumIterations ( HYPRE_SStructSolver solver,
-                                                 HYPRE_Int *num_iterations );
-HYPRE_Int HYPRE_SStructSysPFMGGetFinalRelativeResidualNorm ( HYPRE_SStructSolver solver,
-                                                             HYPRE_Real *norm );
+HYPRE_Int hypre_SStructPVectorSetRandomValues ( hypre_SStructPVector *pvector, HYPRE_Int seed );
+HYPRE_Int hypre_SStructVectorSetRandomValues ( hypre_SStructVector *vector, HYPRE_Int seed );
+HYPRE_Int hypre_SStructSetRandomValues ( void *v, HYPRE_Int seed );
```

Comment:
These 3 should be in `sstruct_mv/protos.h`

Reply:
I'll check